include/materials/linear_elastic.h

```diff
@@ -77,6 +77,16 @@ class LinearElastic : public Material<Tdim> {
   double poisson_ratio_{std::numeric_limits<double>::max()};
   //! Bulk modulus
   double bulk_modulus_{std::numeric_limits<double>::max()};
+  //! S-Wave Velocity
+  double s_wave_velocity_{std::numeric_limits<double>::max()};
+  //! P-Wave Velocity
+  double p_wave_velocity_{std::numeric_limits<double>::max()};
+  //! Layer Thickness
+  double layer_thickness_{std::numeric_limits<double>::max()};
+  //! P Spring Coefficient
+  double p_spring_coeff_{std::numeric_limits<double>::max()};
+  //! S Spring Coefficient
+  double s_spring_coeff_{std::numeric_limits<double>::max()};
```

Comment:
Remove all these lines as it is not good to store layer thickness in the material. Should be computed at the nodes.